apps/team-awesome-web/src/components/time-entries/TimeEntries.tsx

```diff
@@ -101,6 +98,16 @@ export const TimeEntries = ({ ...props }: TimeEntriesProps) => {
     }
   };
 
+  // Inital load
+  useEffect(() => {
+    setTimeEntries(props.timeEntries);
+  }, []);
```

Comment:
Consider removing comments and renaming the variables to make your code clear. E.g. `initialTimeEntries` instead of `props.timeEntries`.

Reply:
_Word of the wise_ was to uphold the `props.timeEntries` solution, opting directly against `initialTimeEntries` so I'll stick with it for now (but definitely noted📝)